IOError: [Errno 2] No such file or directory: './opensfm/reconstruction.json'

While rerunning same job on a 16 CPU instance, received the same error. Previously ran the job on a 8 CPU instance without issue. Prior to running the job on the 16 CPU instance the only directory in place was the images dir.

What gives?

ERROR MESSAGE
2016-11-30 17:11:00,072 Matching DSC01982_geotag.JPG - 388 / 388
2016-11-30 17:11:00,363 reading features
2016-11-30 17:11:10,086 Merging features onto tracks
2016-11-30 17:11:10,086 Good tracks: 0
2016-11-30 17:11:10,367 0 partial reconstructions in total.
Traceback (most recent call last):
File "/root/OpenDroneMap/SuperBuild/src/opensfm/bin/opensfm", line 34, in 
command.run(args)
File "/root/OpenDroneMap/SuperBuild/src/opensfm/opensfm/commands/mesh.py", line 22, in run
reconstructions = data.load_reconstruction()
File "/root/OpenDroneMap/SuperBuild/src/opensfm/opensfm/dataset.py", line 336, in load_reconstruction
with open(self.__reconstruction_file(filename)) as fin:
IOError: [Errno 2] No such file or directory: '/d01/projects/160613_BMM_M4-1608090019_odm/opensfm/reconstruction.json'
[ERROR] quitting cause:
PYTHONPATH=/root/OpenDroneMap/SuperBuild/install/lib/python2.7/dist-packages /root/OpenDroneMap/SuperBuild/src/opensfm/bin/run_all /d01/projects/160613_BMM_M4-1608090019_odm/opensfm
returned with code 256.

--- a/scripts/odm_app.py
+++ b/scripts/odm_app.py
@@ -139,7 +139,7 @@ class ODMApp(ecto.BlackBox):
         # create odm georeference
         connections += [self.tree[:] >> self.georeferencing['tree'],
                         self.args[:] >> self.georeferencing['args'],
-                        self.dataset['photos'] >> self.georeferencing['photos'],
+                        self.resize['photos'] >> self.georeferencing['photos'],
                         self.texturing['reconstruction'] >> self.georeferencing['reconstruction']]
 
         # create odm orthophoto

